### PR TITLE
[SW-1142] Smart contract address is displaying as aeternity when Tx is in pending status on multichain branch

### PR DESCRIPTION
**Pull request by pstachel**

### CODE REVIEW

File: src/popup/components/TransactionDetailsBase.vue [hunk 1]
```diff
@@ -53,7 +53,7 @@
         <slot name="tip-url" />
 
         <DetailsItem
-          v-if="contractId"
+          v-if="contractId && contractId.startsWith('ct_')"
```
Comment (CedrikNikita) [inline src/popup/components/TransactionDetailsBase.vue:56]: Maybe it will be better to use `Ecnoded.ContractAddress` to compare with?

Comment (peronczyk) [reply src/popup/components/TransactionDetailsBase.vue:56]: We have the `isContract` helper function located in `src/protocols/aeternity/helpers/index.ts` for this purpose.

Comment (pstachel) [reply src/popup/components/TransactionDetailsBase.vue]: Fixed

**Comment by Liubov-crypto** [comment]
LGTM

**Review by CedrikNikita — APPROVED** [review]

**Review by peronczyk — APPROVED** [review]

**Pull request merged**